phpunit.xml.dist

```diff
@@ -30,4 +30,8 @@
         </whitelist>
     </filter>
 
+    <php>
+        <ini name="precision" value="8"/>
+    </php>
```

Comment:
> And about splitting this commit in several commits: @core23 told me to squash my commits... ;(

Did he ? I think I made a few mind change about this recently, maybe this was some time ago ? The problem with squashed commits, is that people tend to stick with short messages, so you loose a lot of information. Having to look for information on github is not very good too, we should be able to understand the software with git alone.